high_fiver.rb

```diff
@@ -0,0 +1,5 @@
+module HighFiver
+  def offer_high_five
+    "High five!"
+  end
+end
```

Comment:
Usually for modules that are going to be included in classes, we use "-able" names, so here it'd be HighFiveable.